src/packages/errors/index.ts

```diff
@@ -0,0 +1,20 @@
+import { setTags } from '@sentry/react';
+import React, { useEffect } from 'react';
+
+import { useAuth } from 'context/AuthContext';
+
+export const SentryEventTagger: React.FC = () => {
```

Comment:
I like the approach and I'd take it a step further.
Because Sentry requires to wrap the app with an error boundary component ([`<Sentry.ErrorBoundary/>`](src/App/index.tsx)), I'd transform `SentryEventTagger` into a wrapper that wraps its children within the Sentry error boundary (and does what it currently does as well). I'd also move the logic for initializing [Sentry](src/initializeLibraries/sentry.ts) into this wrapper as well, so all the logic for Sentry is contained within the `errors` package. That's the approach I took with the `analytics` [package](src/packages/analytics/index.tsx).